src/providers/aptos.ts

```diff
@@ -50,6 +63,16 @@ export default class AptosProvider extends BloctoProvider implements AptosProvid
 
     this.appId = appId || process.env.APP_ID;
     this.server = server || defaultServer || process.env.SERVER || '';
+
+    const session: Session | null = getItemWithExpiry<Session>(this.sessionKey, {});
+
+    if (session?.chainId && session.chainId === chainId) {
```

Comment:
can just write `session?.chainId === chainId` as chainId was checked previously and can't be nul